### PR TITLE
Replace hard-coded bucket name with a variable

### PR DESCRIPTION
**Pull request by dav3r**
# 

## 🗣 Description
This PR adds a variable for the name of the bucket containing the Nessus Debian package file and sets it to a useful default value (the formerly hard-coded bucket name).

I also documented the variables and their default values in the README.

## 💭 Motivation and Context
Downstream users of this module (namely [`nessus-packer`](https://github.com/cisagov/nessus-packer)) require the ability to pass in different bucket names depending on the environment (e.g. one bucket for Production and a different one for Staging).

## 🧪 Testing
I overrode the default bucket name (`package_bucket: "cisa-cool-third-party-staging"` in `vars/main.yml`) and successfully ran a local `molecule test` to confirm that the module could install Nessus from the non-default bucket.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: vars/main.yml [hunk 1]
```diff
@@ -1,10 +1,13 @@
 ---
```
Comment (jsf9k) [inline vars/main.yml]: You want to delete this file.  vars are essentially variables that are very hard to override.

**Review by jsf9k — CHANGES_REQUESTED** [review]
Please delete the `vars/main.yml` file, then I will approve.

File: vars/main.yml (hunk 1 above)
Comment (jsf9k) [reply vars/main.yml]: https://codereviewvideos.com/course/ansible-tutorial/video/variable-precedence-where-to-put-your-role-vars

Comment (dav3r) [reply vars/main.yml]: Swift, decisive action taken in https://github.com/cisagov/ansible-role-nessus/pull/9/commits/7ed520a8082e97677e528c2ff6851934971d4942.

**Review by jsf9k — APPROVED** [review]

**Review by felddy — APPROVED** [review]
👍

**Pull request merged**